platform/mk2/hal/gps_skytraq/gps_device_skytraq.c

```diff
@@ -29,9 +29,10 @@
 #define MESSAGE_TYPE_BINARY		2
 #define GNSS_NAVIGATION_MODE		GNSS_NAVIGATION_MODE_AUTOMOBILE
 
-#define BAUD_RATE_COUNT 2
+#define BAUD_RATE_COUNT 3
 #define BAUD_RATES 	{			\
 		{921600, 8},			\
+		{115200, 5},            	\
```

Comment:
I believe the GPS always powers up in 9600, so I don't think 115200 is needed here.